csv/write.go

```diff
@@ -6,18 +6,23 @@ import (
 	"io"
 
 	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/apache/arrow/go/v13/arrow/array"
 	"github.com/apache/arrow/go/v13/arrow/csv"
+	"github.com/apache/arrow/go/v13/arrow/memory"
 	"github.com/cloudquery/plugin-sdk/v3/schema"
 )
 
 func (cl *Client) WriteTableBatch(w io.Writer, table *schema.Table, records []arrow.Record) error {
-	writer := csv.NewWriter(w, table.ToArrowSchema(),
+	newSchema := convertSchema(table.ToArrowSchema())
+	writer := csv.NewWriter(w, newSchema,
```

Comment:
not necessarlity related to this PR but I think we should copy the `csv` directory from arrow to here under `internal` so we have better control of this (just keep the license and copyright).